ui/ts/models/log.ts

```diff
@@ -77,13 +76,22 @@ module Models {
         return url;
       }
 
-      constructor() {
+      constructor(nodeId?: string) {
         this.level(Utils.Format.Severity(2));
         this.max(null);
         this.startTime(null);
         this.endTime(null);
         this.pattern(null);
-        this.node(null);
+        this.node(nodeId);
+        // Add the QueryCache after initialization in order to not fetch without
```

Comment:
For the most part, we actually want it to be eager.  For node/store status, ts values, and more, it doesn't require an id in the url.
I'm adding the ability to have it wait for an init call in a subsequent pr right now.